<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>ft_config_dialog</class>
    <widget class="QDialog" name="ft_config_dialog">
        <property name="windowModality">
            <enum>Qt::ApplicationModal</enum>
        </property>
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>294</width>
                <height>151</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>146</width>
                <height>109</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>294</width>
                <height>151</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>FieldTrip Konfiguration - MidiBrain</string>
        </property>
        <property name="modal">
            <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <spacer name="verticalSpacer">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>40</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                        <widget class="QLabel" name="label">
                            <property name="text">
                                <string>Host-Name:</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <spacer name="horizontalSpacer">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QLineEdit" name="hostname_input">
                            <property name="inputMask">
                                <string/>
                            </property>
                            <property name="text">
                                <string>localhost</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                        <widget class="QLabel" name="label_2">
                            <property name="text">
                                <string>Port-Nr.:</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <spacer name="horizontalSpacer_2">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QLineEdit" name="portnr_input">
                            <property name="inputMask">
                                <string>9999</string>
                            </property>
                            <property name="text">
                                <string>1972</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QPushButton" name="connect_button">
                    <property name="text">
                        <string>Verbinden</string>
                    </property>
                </widget>
            </item>
            <item>
                <spacer name="verticalSpacer_2">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>40</height>
                        </size>
                    </property>
                </spacer>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
